src/lib.rs

```diff
@@ -55,6 +56,18 @@ impl FixedBitSet
         }
     }
 
+    /// Shrink capacity to **bits**.  If **bits** is greater than length, has no effect.
+    pub fn shrink(&mut self, bits: usize) {
+        if bits >= self.length {
+            return;
+        }
+        let (mut blocks, rem) = div_rem(bits, BITS);
+        blocks += (rem > 0) as usize;
+        self.data.truncate(blocks+1);
```

Comment:
I wouldn't block merging it for this, but we should try to use spaces around operators consistently, so `blocks + 1` here.